lib/gears/watcher.lua

```diff
@@ -0,0 +1,733 @@
+--- Fetch information at a specific interval.
+--
+-- @author Emmanuel Lepage-Vallee &lt;elv1313@gmail.com&gt;
+-- @copyright 2020 Emmanuel Lepage-Vallee
+-- @classmod gears.watcher
+
+local capi = {awesome = awesome}
+local protected_call = require("gears.protected_call")
+local gtimer = require("gears.timer")
+local gtable = require("gears.table")
+local lgi = require("lgi")
+local Gio = lgi.Gio
+local GLib = lgi.GLib
+
+local module = {}
+
+-- This is awful.util.shell
+module._shell = os.getenv("SHELL") or "/bin/sh"
```

Comment:
...but is useless. The point of `awful.util.shell` is being configurable, I guess the `_` in `_shell` here means that this is not supposed to be configurable here.

Reply:
facepalm... fixed